shopfloor/services/cluster_picking.py

```diff
@@ -406,9 +411,10 @@ def _data_move_line(self, line, **kw):
         data["batch"] = self.data.picking_batch(batch)
         data["picking"] = self.data.picking(picking)
         data["postponed"] = line.shopfloor_postponed
-        data["product"]["qty_available"] = product.with_context(
-            location=line.location_id.id
-        ).qty_available
+        if "no_qty_available" not in kw or not kw.get("no_qty_available"):
+            data["product"]["qty_available"] = product.with_context(
+                location=line.location_id.id
+            ).qty_available
```

Comment:
Is it possible to avoid double negations ? `not no_qty_available` is not really intuitive to read.
For instance, explicitely add a named argument `with_qty_available` which defaults to `True`.
When you don't need this information in the data, we can just set `with_qty_available=False`.
Then you can do this, which IMO is easier to read. and less bug prone.
```suggestion
        if with_qty_available:
            data["product"]["qty_available"] = product.with_context(
                location=line.location_id.id
            ).qty_available
```
What do you think ?